### PR TITLE
Update README, add LICENSE

### PR DESCRIPTION
**Pull request by Hwesta**
Update README to be more welcoming on github, using advice from https://github.com/ddbeck/readme-checklist/blob/master/checklist.md

Add a LICENSE file (copied from Archivematica) to make the licensing clear.

Similar to artefactual/archivematica#461

### CODE REVIEW

File: fpr/urls.py [hunk 1]
```diff
@@ -5,7 +5,7 @@
 
 urlpatterns = [
     url(r'^$', views.home, name='fpr_index'),
-    url(r'^(?P<category>format|formatgroup|idrule|idcommand|fprule|fpcommand)/(?P<uuid>' + UUID_REGEX + ')/toggle_enabled/$', 'toggle_enabled',
+    url(r'^(?P<category>format|formatgroup|idrule|idcommand|fprule|fpcommand)/(?P<uuid>' + UUID_REGEX + ')/toggle_enabled/$', views.toggle_enabled,
```
Comment (Hwesta) [inline fpr/urls.py:8]: Ignore this - it's already merged but accidentally got included in the PR.

**Comment by sevein** [comment]
👍

**Comment by jhsimpson** [comment]
:+1:  looks great.  Should the readme.md be updated after PR32 is merged?

**Comment by Hwesta** [comment]
> Should the readme.md be updated after #32 is merged? 

To include information on the translation? Or something else?

**Comment by Hwesta** [comment]
Cherry-picked to qa/1.x as a1fd5d6538cfb0aa2198755ba2d2cb9d4084e76f 75412def23e8c5146e8de7df2407a91e60f5501b

**Pull request merged**